scripts/setup_roboschool.sh

```diff
@@ -0,0 +1,37 @@
+echo "Make sure this script is running under garage conda environment"
+
+# install dependency
+if [[ "$(uname)" == "Darwin" ]]; then
+	brew install cmake tinyxml assimp ffmpeg
+	brew reinstall boost-python3 --without-python --with-python3 --build-from-source
+	conda install qt
+	export PKG_CONFIG_PATH=$(dirname $(dirname $(which python)))/lib/pkgconfig
+elif [[ "$(expr substr $(uname -s) 1 5)" == "Linux" ]]; then
+	sudo apt-get install -y ffmpeg pkg-config qtbase5-dev libqt5opengl5-dev libpython3.5-dev \
+	  libassimp-dev libboost-python-dev libtinyxml-dev
+fi
+
+# clone roboschool and bullet
+read -e -p "Please enter a path to clone roboschool and bullet3 repo [eg: /home/name/code]: " PARENT_DIR
+eval PARENT_DIR=${PARENT_DIR}
```

Comment:
if the variable is "~/code", it will be translated to "/home/name/code". without this line, `git` will create a "~" directory in working directory. now its deleted because of using temporary directory.